helm/sealed-secrets/crds/bitnami.com_sealedsecrets.yaml

```diff
@@ -118,9 +120,3 @@ spec:
         type: object
     served: true
     storage: true
-status:
-  acceptedNames:
-    kind: ""
-    plural: ""
-  conditions: []
-  storedVersions: []
```

Comment:
Is this required?

Reply:
This was removed by `controller-gen` version `0.9.2`.